README.md

```diff
@@ -1 +1,57 @@
-# microshift
+# Microshift
+Microshift (aka "µShift") is OpenShift<sup>1</sup> Kubernetes in a small form factor and optimized for edge computing.
+
+Edge devices deployed out in the field pose very different operational, environmental, and business challenges from those of cloud computing. These motivate different engineering trade-offs for Kubernetes at the far edge than for cloud or near-edge scenarios. Microshift's design goals cater to this:
+
+* make frugal use of system resources (CPU, memory, network, storage, etc.),
+* tolerate severe networking constraints,
+* update (resp. roll back) securely, safely, speedily, and seamlessly (without disrupting workloads), and
+* build on and integrate cleanly with edge-optimized OSes like Fedora IoT and RHEL for Edge, while
+* providing a consistent development and management experience with standard OpenShift.
+
+We believe these properties should also make Microshift a great tool for other use cases such as Kubernetes applications development on resource-constrained systems, scale testing, and provisioning of lightweight Kubernetes control planes.
+
+**Note: Microshift is still early days and moving fast. Features are missing. Things break. But you can still help shape it, too.**
+
+<sup>1) more precisely [OKD](https://www.okd.io/), the Kubernetes distribution by the OpenShift community</sup>
+
+## Using Microshift
+To give Microshift a try, simply install a recent test version (we don't provide stable releases yet) using:
+```
+curl -sfL https://raw.githubusercontent.com/redhat-et/microshift/main/install.sh | sh -
+```
+
+This will install Microshift's dependencies (CRI-O), install it as a systemd service and start it.
+
+For convenience, the script copies the kubeconfig to $HOME/.kube/config, too, so after installation you should be able to access your cluster directly:
```

Comment:
Per my comment in https://github.com/redhat-et/microshift/pull/99#discussion_r652952649,  we shouldn't obliterate user's kubeconfigs.

Reply:
Updated this accoring to resolution for #99.